eva-tools/src/main/java/embl/ebi/variation/eva/vcfDump/VariantExporter.java

```diff
@@ -228,8 +428,58 @@ public VariantExporter(CellBaseClient cellbaseClient) {
 
     }
 
+    private Set<String> getChromosomes(Map<String, VCFHeader> headers, List<String> studyIds, QueryOptions options) {
+        Set<String> chromosomes;
+
+        List<String> regions = options.getAsStringList(VariantDBAdaptor.REGION);
+        if (regions.size() > 0) {
+            chromosomes = getChromosomesFromRegionFilter(regions);
+        } else {
+            chromosomes = getChromosomesFromCellbaseREST();
+            if (chromosomes == null || chromosomes.isEmpty()) {
+                chromosomes = getChromosomesFromVCFHeader(headers, studyIds);
+            }
+        }
+
+        return chromosomes;
+    }
+
+    private Set<String> getChromosomesFromRegionFilter(List<String> regions) {
+        return regions.stream().map(r -> r.split(":")[0]).collect(Collectors.toSet());
+    }
+
+    private Set<String> getChromosomesFromCellbaseREST() {
+        // call cellbase chromosomes WS
+        RestTemplate restTemplate = new RestTemplate();
+        ParameterizedTypeReference<QueryResponse<QueryResult<CellbaseChromosomesWSOutput>>> responseType =
+                new ParameterizedTypeReference<QueryResponse<QueryResult<CellbaseChromosomesWSOutput>>>() {};
+        // TODO: property file or command line option for cellbase rest URL? OPENCGA?
+        ResponseEntity<QueryResponse<QueryResult<CellbaseChromosomesWSOutput>>> wsOutput =
+                restTemplate.exchange("http://bioinfo.hpc.cam.ac.uk/cellbase/webservices/rest/v3/hsapiens/genomic/chromosome/all",
```

Comment:
Does the deployment at EBI retrieve different information? Otherwise we should prefer www.ebi.ac.uk as hostname.